kitsune/wiki/forms.py

```diff
@@ -276,6 +276,23 @@ def save(self, creator, document, based_on_id=None, base_rev=None,
         return new_rev
 
 
+class DraftRevisionForm(forms.ModelForm):
+    class Meta(object):
+        model = DraftRevision
+        fields = ('keywords', 'summary', 'content', 'slug', 'title',
+                  'based_on')
+
+    def save(self, user, parent_doc, doc_data):
+        """save the draft revision and return the draft revision"""
+        locale = doc_data["locale"]
+        doc_data.pop("locale", None)
```

Comment:
`.pop` is used to remove *and return* a value from a dict. This is better written as `locale = doc_data.pop('locale')`.